assets/scripts/dialogs/GeotagDialog.jsx

```diff
@@ -302,6 +314,7 @@ class GeotagDialog extends React.Component {
               zoomControl={false}
               zoom={this.state.zoom}
               onClick={this.handleMapClick}
+              useFlyTo
```

Comment:
I could've sworn I wrote 
``` useFlyTo={true} ```
Here. Maybe I mistyped or it got auto-linted? Is either one considered more of a React best practice? Normally I'd prefer being more explicit..

Reply:
You probably did, and it got auto-linted. We are applying the [react/jsx-boolean-value](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) rule here. However, we didn't set this one ourselves, it comes from the [`eslint-config-standard-jsx`](https://github.com/standard/eslint-config-standard-jsx/blob/master/eslintrc.json) package (which is imported by `eslint-config-standard`).

I actually agree with you here. It makes it easier to toggle a `true` value to `false` (or vice versa) without having to fiddle with the syntax. Furthermore, the [React docs](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true) has expressed their own preference:

> In general, we don’t recommend not passing a value for a prop, because it can be confused with the ES6 object shorthand {foo} which is short for {foo: foo} rather than {foo: true}. This behavior is just there so that it matches the behavior of HTML.

This means people learning "best practices" (or at least, normalized practices) in React see a contradiction between React docs and our implementation of it, and we have no good rationale other than "this is what the "standard" eslint rules enforce." Normally I would hesitate to make a bikeshed argument against standard's rules but this might be a warranted case.

If you agree, I can see two possible options forward (not necessarily mutually exclusive):

1. We make a rule override in `.eslintrc` and document our rationale (easy)
2. We make a case to Feross to change his rule, citing React's preference (might take a while and would require a major version bump on his end)

Reply:
Hmm, interesting! 

I'll have to give this some thought. I wonder where else in the streetmix codebase this might be affected. There are also other eslint standards that exist and are more restrictive, which opens up another can of worms but could actually be helpful guides for refactoring. 

e.x. i swapped this in real quick: 

```  
"extends": [
"eslint:recommended", "plugin:react/recommended"
  ],
```

and got a bunch of 'variable assigned but not used' errors.

Reply:
PR with tweaked rule + fixes here: https://github.com/streetmix/streetmix/pull/1921

Reply:
Sounds like we can consider the eslint recommended rules later too. #1921 is merged, so you should be able to update this PR with the syntax you'd expect!